src/store/edit-profile/index.ts

```diff
@@ -0,0 +1,38 @@
+import { createAction, createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+export enum SagaActionTypes {
+  EditProfile = 'profile/edit',
+}
+
+export type EditProfileState = {
+  loading: boolean;
+  errors: string[];
+  changesSaved: boolean;
```

Comment:
:thinking: would the `loading` and `changesSaved` combo be better represented as a `state` enum? Like: `NONE`, `IN_PROGRES`, `SUCCESS` or something along those lines.

Reply:
done